src/diff/mount.js

```diff
@@ -237,7 +238,13 @@ export function mountChildren(parentInternal, children, parentDom, startDom) {
 		vnode = children[i];
 
 		// account for holes by incrementing the index:
-		if (vnode == null || vnode === true || vnode === false) continue;
+		if (
+			vnode == null ||
+			vnode === true ||
+			vnode === false ||
+			typeof vnode === 'function'
```

Comment:
Should we do this typeof once and reuse it's result here and line 249?

Same for patch

Reply:
Just checked and storing the result in a variable is 10b larger. I think for now it's fine to leave it as is.